x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -562,6 +564,46 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     return await this.baseClient.removeReferencesTo(type, id, options);
   }
 
+  public async openPointInTimeForType(
+    type: string | string[],
+    options: SavedObjectsOpenPointInTimeOptions
+  ) {
+    try {
+      const args = { type, options };
+      await this.ensureAuthorized(type, 'open_point_in_time', options?.namespace, { args });
+    } catch (error) {
+      this.auditLogger.log(
+        savedObjectEvent({
+          action: SavedObjectAction.OPEN_POINT_IN_TIME,
+          error,
+        })
+      );
+      throw error;
+    }
+
+    const pit = await this.baseClient.openPointInTimeForType(type, options);
+
+    this.auditLogger.log(
```

Comment:
nit: let's move this audit log event before the call to `this.baseClient.openPointInTimeForType` and add `outcome: EventOutcome.UNKNOWN,` to the payload

Reply:
done